x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/ExtractSubstring.java

```diff
@@ -14,9 +14,9 @@
 import static org.elasticsearch.common.Strings.hasLength;
 import static org.elasticsearch.xpack.ql.util.StringUtils.EMPTY;
 
-final class StringUtils {
+final class ExtractSubstring {
```

Comment:
Similar to the code above - is there an existing method with similar functionality that could be used instead of this method?